.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+name: GitHub Actions CI
+on: [push]
+jobs:
+  tests:
+    runs-on: macOS-latest
+    steps:
+    - name: Set up Git repository
+      uses: actions/checkout@master
+
+    - name: Set up Ruby
+      run: |
+        brew uninstall --ignore-dependencies --force $(brew list)
+        brew install rbenv
+        RBENV_VERSION="2.0.0-p481"
+        rbenv install "$RBENV_VERSION"
+        export PATH="$HOME/.gem/ruby/2.3.0/bin:$PATH"
```

Comment:
I don't think `export` works across steps.

Reply:
@reitermarkus Thanks! I guess we'll find out soon 😆 🍿